### PR TITLE
NAS-102237 / 11.3 / Expose method to restart UI

### PR DESCRIPTION
**Pull request by sonicaj**
This commit exposes a method in system.general service which can be used to restart UI after changes in UI have been made like cert changes etc. The method restarts nginx and django as soon as it is called.

### CODE REVIEW

File: src/middlewared/middlewared/plugins/system.py [hunk 1]
```diff
@@ -1002,6 +1002,14 @@ def _get_index(country_columns, column):
 
         return await self.config()
 
+    @accepts()
+    async def ui_restart(self):
+        """
+        Restart nginx server and django to reflect latest UI settings.
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py]: Lets not mention technologies. Just say HTTP server.

Comment (a-stjohn) [inline src/middlewared/middlewared/plugins/system.py]:
```suggestion
        Restart nginx server and django to show latest UI settings.
```

File: src/middlewared/middlewared/plugins/system.py [hunk 2]
```diff
@@ -1002,6 +1002,14 @@ def _get_index(country_columns, column):
 
         return await self.config()
 
+    @accepts()
+    async def ui_restart(self):
+        """
+        Restart HTTP server to show latest UI settings.
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py]: to show seems wrong here. `to use` maybe? There is nothing to show

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/system.py]: @freenas/docs-team please advise

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/system.py]: And i agree @william-gr

Comment (a-stjohn) [reply src/middlewared/middlewared/plugins/system.py]: `to use` works.

**Comment by wblock** [comment]
It really means "So UI settings changes take effect."

**Pull request merged**